packages/manager/modules/navbar/src/index.less

```diff
@@ -25,6 +25,85 @@
   }
 }
 
+.hub-navbar-menu {
+  @navbar-menu-background-color: #f1f5f9;
+  @navbar-menu-content-background-color: #ffffff;
+  @navbar-menu-content-hover-background-color: #bef1ff;
+  @navbar-alert-background-color: #c11b1b;
+  @navbar-menu-content-color: #113f6d;
+  @navbar-menu-content-shadow-color: #e3e6e8;
+
+  top: 2.85rem;
+
+  .oui-navbar-list {
+    background-color: @navbar-menu-background-color;
```

Comment:
Maybe this style overwrite should be handled more globally for all the hub app (maybe even shared with public-cloud as a `manager-light-theme`or something like that)